MeshLib/MeshSubset.h

```diff
@@ -91,6 +91,11 @@ class MeshSubset
         return _msh.getElements().cend();
     }
 
+    const Mesh &getMesh() const
+    {
+        return _msh;
+    }
```

Comment:
is this method necessary? from what i gather you could access the mesh from the general mesh vector using the mesh id provided in this class? i am not sure, though.